cmd/evmosd/root.go

```diff
@@ -94,8 +93,9 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 
 			// override the app and tendermint configuration
 			customAppTemplate, customAppConfig := initAppConfig()
+			customTMConfig := initTendermintConfig()
 
-			err = sdkserver.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig)
+			err = sdkserver.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig, customTMConfig)
 			if err != nil {
 				return err
 			}
```

Comment:
you need to replace everything below this line to just return (no need to set the context again)

```
			// override the app and tendermint configuration
			customAppTemplate, customAppConfig := initAppConfig()
			customTMConfig := initTendermintConfig()

			return server.InterceptConfigsPreRunHandler(cmd, customAppTemplate, customAppConfig, customTMConfig)
```

Reply:
@faddat

Reply:
Ok

Reply:
changed